### ISSUE TITLE
[Bug][SF][2.6.0][FlexibleColumnLayout][A11y][Expected reply Time: ASAP] Resizable column separator does not meet the minimum target size

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents-react#6866]
### Describe the bug

Resizable column separator does not meet the minimum target size

<img width="1542" alt="Image" src="https://github.com/user-attachments/assets/6987d15d-1f2b-436a-ab63-2c98d92ecb0b" />

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-flexiblecolumnlayout--docs&args=disableResizing:!true

### Reproduction steps

Check the size of separator

### Expected Behaviour

The keyboard focus should be visible on the separator in all modes.

### UI5 Web Components for React Version

2.6.0

### UI5 Web Components Version

2.6.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
